src/devices_models/devices/common/get_time_series.jl

```diff
@@ -9,7 +9,7 @@ function get_time_series(
     time_steps = model_time_steps(psi_container)
     has_forecasts = PSY.has_forecasts(device)
     if !has_forecasts
-        @warn("device $(typeof(device)) $(PSY.get_name(device)) has forecasts = $(PSY.has_forecasts(device))")
+        @warn "$(summary(device)) does not have forecasts)"
```

Comment:
Per my earlier comment, is it appropriate to log this warning when `use_forecast_data` is false?